sunpy/net/dataretriever/client.py

```diff
@@ -64,6 +64,8 @@ class GenericClient(BaseClient):
     pattern = None
     # Set of required 'attrs' for client to handle the query.
     required = {a.Time, a.Instrument}
+    # Define keywards a client needs to pass to enqueue_file
+    enqueue_file_kwargs = {}
```

Comment:
We should probably document this new feature somewhere. Maybe the Fido dev docs?

Reply:
Can do.

Reply:
Should we open an issue for this?

Reply:
I forgot to do this change before it was merged. I will open a PR now.

Reply:
THE TYPO TOO